include/ibamr/IBFEMethod.h

```diff
@@ -821,8 +821,19 @@ class IBFEMethod : public IBStrategy
     void putToDatabase(SAMRAI::tbox::Pointer<SAMRAI::tbox::Database> db) override;
 
     /*!
-     * Write the equation_systems data to a restart file in the specified directory.
-     */
+    *Restart files for the SAMRAI data is written using the built in RestartManager, but special restart files
```

Comment:
It would make more sense to just write "for technical reasons this class does not use SAMRAI's RestartManager" etc.